### ISSUE TITLE
AMBER BAF file contains unknown character

### ISSUE THREAD

**Issue by pdiakumis** [hartwigmedical/hmftools#10]
Just wanted to report this little bug I'm getting when running PURPLE. It's stumbling onto an unknown character in the BAF file generated by AMBER. I ran it twice and it re-appeared, so it's reproducible at least. Not sure if it's to do with the mpileup files generated by sambamba - I checked in them around the area just in case. Do you think it would be safe to just delete the problematic line and continue on?

Many thanks - Peter

![screen shot 2018-08-08 at 13 13 05](https://user-images.githubusercontent.com/29562861/43814277-e340c728-9b0c-11e8-88cf-a3b11b78295d.png)

**Comment by pdiakumis** [comment]
Yep, running Amber: `version=1.5`, `build.date=2018-02-14 06:06`

Attaching pileups with 3 before/after context lines:

```
grep -P -C3 "^16\t76595025" HCC2218BL.mpileup > HCC2218BL_bad.mpileup
grep -P -C3 "^16\t76595025" HCC2218C.mpileup > HCC2218C_bad.mpileup
```
I'll delete the offending line and continue for now. Many thanks for your help!

[bad_mpileups.zip](https://github.com/hartwigmedical/hmftools/files/2268873/bad_mpileups.zip)

**Comment by jonbaber** [comment]
Thanks for providing the files. 

I have released [version 1.6](https://github.com/hartwigmedical/hmftools/releases/tag/amber-v1-6) of amber where this is fixed. 

Cheers, Jon
